config/swagger.yaml

```diff
@@ -1933,6 +1935,7 @@ definitions:
       - target
       - time
       - version
+      - info
```

Comment:
I see same Swagger type used for response of [pending path](https://github.com/aeternity/protocol/blob/0d178475a5794fa7bbc623c4a0f4c163123d580d/node/api/mining_api_usage.md#aeternity-node-modes) so ok.